README.md

```diff
@@ -110,6 +103,12 @@ $ gdm brew
   ...
 ```
 
+### Restore
+
+The `gdm restore` command is deprecated as of the release of Go 1.9.
```

Comment:
Can you clarify what deprecated means in this context? It's not going to be a huge issue for us to switch to `gdm vendor` in the future, but it will be painful to have to backport tooling on old versions. I'd prefer to see `gdm restore` stay a valid command forever. If it's only there for backwards compatibility, I think it's completely fine to remain a hidden command that does not show up during `gdm help`.

Reply:
fair enough, I can leave it in and just describe the difference between the two commands